src/client/redux/store/configureStore.ts

```diff
@@ -0,0 +1,9 @@
+import { createStore, applyMiddleware, compose } from 'redux';
+import rootReducer from '../reducers';
+import { IAppState } from '../reducers';
```

Comment:
use single import statement for importing from the same file

PS there is a tslint rule for this, please add tslint to project